go/libraries/doltcore/table/typed/json/reader.go

```diff
@@ -111,7 +111,12 @@ func (r *JSONReader) ReadSqlRow(ctx context.Context) (sql.Row, error) {
 		return nil, io.EOF
 	}
 
-	return r.convToSqlRow(metaRow.Value.(map[string]interface{}))
+	mapVal, ok := metaRow.Value.(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("unexpected JSON format received, expected format: { \"rows\": [ json_row_objects... ] } ")
```

Comment:
Good error message! 👏 This should be helpful for customers to understand the input format. 

Could you also add some more docs to the `dolt table import` command so that this expected input format is described there, too? I didn't see anything in the docs yet describing what we expect for a JSON input file. Hopefully putting a note about that format in the docs will help customers more easily understand how to use it.